pkg/nfd-worker/nfd-worker.go

```diff
@@ -303,14 +295,6 @@ func (w *nfdWorker) Run() error {
 		return err
 	}
 
-	// Create watcher for TLS certificates
-	w.certWatch, err = utils.CreateFsWatcher(time.Second, w.args.CaFile, w.args.CertFile, w.args.KeyFile)
```

Comment:
Drop CaFile and friends from `args`, too (and the flags from main.go)

Reply:
done

Reply:
Hmm, the stuff is still in `Args`. Alson the `Server` and `ServerNameOverride` should be dropped